gather_data: merge job and test gathering (#58)
tarantool/multivac: README.md

$ ./multivac/last_seen.py --branch master --branch sample-branch --format html
```

### gather_data.py

SYNOPSIS
    
    ./multivac/gather_data.py [OPTIONS]

DESCRIPTION
    

>     Get data about every completed job and failed test and group them by job ID.

Author: done

    See the parameters it collects on the
    [website](https://www.tarantool.io/en/dev/multivac/gather_data/).

OPTIONS
    
    --tests, -t

>             Collect data about tests. Data collects with 
>             `multivac/sensors/test_status.py` from test logs. Without this
>             option the script will collect data only about workflows.

Reviewer: At this level it's irrelevant which class is used to collect test data.

```suggestion
            Collect data about test failures from test logs. Without this
            option the script will collect data only about workflows.
```

Author: done

EXAMPLE
    

>     Collect data about jobs and tests started a week ago and earlier and put
>     them to the InfluxDB:

Reviewer: 'earlier' means a week ago or before that.

data is singular, so them→'it' or 'this data'

```suggestion
    Collect data about jobs and tests which started a week ago or later,
    then put this data to InfluxDB:
```

Author: done

```console

> $ ./multivac/gather_data.py -t --since 7d --format influxdb

Reviewer: I like this example!
